<launch>

  <node pkg="prophesee_ros2_driver" exec="prophesee_ros2_viewer" name="prophesee_ros2_viewer" output="screen">
    <param name="camera_name" value="camera" />
    <param name="show_cd" value="true" />

    <!-- Time interval to display events up to the current time, in us -->
    <param name="display_accumulation_time" value="5000" />
  </node>

</launch>
